services/supervisord/supervisord.go

```diff
@@ -496,8 +497,8 @@ func (s *Service) saveConfigAndReload(name string, cfg []byte) (bool, error) {
 	return true, nil
 }
 
-// UpdateConfiguration updates Prometheus, Alertmanager, and qan-api2 configurations, restarting them if needed.
-func (s *Service) UpdateConfiguration(settings *models.Settings) error {
+// UpdateConfiguration updates Prometheus, Alertmanager, Grafana and qan-api2 configurations, restarting them if needed.
+func (s *Service) UpdateConfiguration(settings *models.Settings, ssoDetails *models.PerconaSSODetails) error {
```

Comment:
aren't soo params a part of settings? why in a separate function param?

Reply:
I store sso details in separate table to avoid sending it to user accidentally as part of settings. Not sure if it is reasonable now. Because one would need to make changes in pmm api to be able to fetch it.

Reply:
So it always will be one record in that table?

Reply:
Yes. @artemgavrilov